configuration/coding_plans.py

```diff
@@ -108,6 +108,24 @@ def get_rqa_coding_plans(pipeline_name):
                    ws_code=CodeSchemes.WS_CORRECT_DATASET.get_code_with_match_value("OXFAM WASH s01e03 Noise Handler"),
                    raw_field_fold_strategy=FoldStrategies.concatenate),
 
+        CodingPlan(raw_field="s01_close_out_raw",
+                   time_field="sent_on",
+                   run_id_field="s01_close_out_run_id",
+                   coda_filename="OXFAM_WASH_s01_Close_Out.json",
```

Comment:
```suggestion
                   coda_filename="OXFAM_WASH_s01_close_out.json",
```
(This is the convention we've fallen into for no particular good reason, no need to change anything now though here)

Reply:
Thanks - will keep as-is for consistency with the rest of the coda file names for this project.